R/Data-methods.R

```diff
@@ -647,18 +764,26 @@ setMethod(
   f = "dose_grid_range",
   signature = signature(object = "Data"),
   definition = function(object, ignore_placebo = TRUE) {
-    assert_flag(ignore_placebo)
+    h_obtain_dose_grid_range(object, ignore_placebo)
+  }
+)
 
-    dose_grid <- if (ignore_placebo && object@placebo && object@nGrid >= 1) {
-      object@doseGrid[-1]
-    } else {
-      object@doseGrid
-    }
 
-    if (length(dose_grid) == 0L) {
-      c(-Inf, Inf)
-    } else {
-      range(dose_grid)
-    }
+## DataOrdinal ----
+
+#' @include Data-methods.R
+#' @rdname dose_grid_range
+#' @description `r lifecycle::badge("experimental")`
```

Comment:
is this needed? I would guess the lifecycle badge applies to the whole .rd page, so not needed here?